src/app/api/tasks/[id]/route.ts

```diff
@@ -0,0 +1,55 @@
+import { NextRequest, NextResponse } from 'next/server';
+import prisma from '../../../../../lib/prismadb';
+import exp from 'constants';
+
+export async function PUT(request: NextRequest, context: { params: any }) {
+  const id = context.params.id;
+  const { title, description, assigneeEmail } = await request.json();
+  const task = await prisma.$transaction(async () => {
+    const task = await prisma.task.update({
+      where: { id },
+      data: {
+        title,
+        description,
+      },
+    });
+    try {
+      assigneeEmail &&
+        (await prisma.assignedTask.update({
+          where: { taskId_assigneeEmail: { taskId: task.id, assigneeEmail } },
```

Comment:
Use the assigneeEmail from the task in the where clause.
Also add check if new assigneeEmail is different from the current task.assigneeEmail before updating

Reply:
i cant use task.assgneeEmail because 
<img width="505" alt="Monosnap route ts — management-system 2024-01-21 19-04-16" src="https://github.com/idoRosen25/management-system/assets/7353245/1a6a33e1-2e32-450c-b7b9-8d01ee5f8e8d">

Reply:
Return the assignee like in the get function and then you will be able to use it

Reply:
Done